app/kudos/templates/shared/kudos_card_hover_content.html

```diff
@@ -16,20 +16,41 @@
 {% endcomment %}
 {% load i18n static kudos_extras  %}
 {% load humanize %}
-<p>Sent {{relation}} 
 
-  {% if relation == 'to' %}
-    <a href="{% url 'profile' kt.username %}">{{kt.username}}</a>
-  {% else %}
-    <a href="{% url 'profile' kt.from_username %}">{{kt.from_username}}</a>
-  {% endif %}
+{% for kt in kudos_group.list %}
+  <div class="d-flex flex-row">
+
+    {% if relation == 'to' %}
+    <img src="/dynamic/avatar/{{ kt.username }}" alt="{{ kt.username }}" class="sender-avatars rounded-circle mr-3" width="45" height="45">
+    <div class="d-flex flex-column">
+      <a href="{% url 'profile' kt.username %}" >{{kt.username}}</a>
+      <time datetime="{{kt.created_on|date:'c'}}" title="{{kt.created_on}}">
+        {{kt.created_on|naturaltime}}
+      </time>
+    </div>
+    {% else %}
+    <img src="/dynamic/avatar/{{ kt.from_username }}" alt="{{ kt.from_username }}" class="sender-avatars rounded-circle mr-3" width="45" height="45">
+    <div class="d-flex flex-column">
+      <a href="{% url 'profile' kt.from_username %}" title="{% trans 'go to profile' %}">{{kt.from_username}}</a>
+      <time datetime="{{kt.created_on|date:'c'}}" title="{{kt.created_on}}">
+        {{kt.created_on|naturaltime}}
+      </time>
+    </div>
+    {% endif %}
 
-{{kt.created_on|naturaltime}}</p>
-<p>
-  <a href="{% if not kt.network == 'mainnet' %}https://{{ kt.network }}.etherscan.io/tx/{{ kt.txid }}{% else %}https://etherscan.io/tx/{{ kt.txid }}{% endif %}" target="_blank" class="d-block mb-1">{{ kt.txid|humanize_address }}</a>
-</p>
+  </div>
 {% if kt.comments_public %}
-  {% trans "Comment:" %}
+<div class="bubble">
   <p>{{kt.comments_public}}</p>
+</div>
```

Comment:
indent by 1

Reply:
are you the new stickler-ci ? hahaa joking, making the fixes here 👍

Reply:
lol lol ! occasionally stickler needs a helping hand :P